actiontext/app/assets/javascripts/actiontext.js

```diff
@@ -846,17 +849,32 @@
       return this.blobUrlTemplate.replace(":signed_id", signedId).replace(":filename", encodeURIComponent(filename));
     }
     get directUploadUrl() {
-      return this.element.dataset.directUploadUrl;
+      return this.delegate.directUploadUrl;
     }
     get blobUrlTemplate() {
-      return this.element.dataset.blobUrlTemplate;
+      return this.delegate.blobUrlTemplate;
+    }
+  }
+  function notify(object, methodName, ...messages) {
+    if (object && typeof object[methodName] == "function") {
+      return object[methodName](...messages);
     }
   }
   addEventListener("trix-attachment-add", (event => {
```

Comment:
Am I misunderstanding something, or is this still tied to Trix? Is it outside of the scope of this PR to make this part of the app work, or should we pull these events from somewhere to make them not Trix-specific?